projects/plugins/boost/app/assets/src/js/pages/index/index.tsx

```diff
@@ -36,6 +37,11 @@ const Index = () => {
 	const hasPremiumCdnFeatures =
 		premiumFeatures.includes( 'image-cdn-liar' ) && premiumFeatures.includes( 'image-cdn-quality' );
 
+	const openCriticalCssLink = () => {
+		recordBoostEvent( 'critical_css_link_clicked', {} );
+		window.open( criticalCssLink, '_blank' );
```

Comment:
No need to handle opening the url in the onClick callback. It's already handled by default, since the anchor has a `href` value.

Reply:
Argh. I thought I had removed that after testing it. You're totally right.